.github/workflows/doc_build.yml

```diff
@@ -0,0 +1,75 @@
+name: Doc Build
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - Linux CI (pytorch-linux-xenial-py3.6-gcc5.4)
+  pull_request:
```

Comment:
this line `pull_request` will just for testing purpose - really need to trigger this workflow to start iterating on the testing, ideally the doc build should only run on master and release after `Linux CI (pytorch-linux-xenial-py3.6-gcc5.4)` is finished.